plugins/kotlin/idea/resources-en/intentionDescriptions/FoldIfToFunctionCallIntention/description.html

```diff
@@ -0,0 +1,5 @@
+<html>
+<body>
+This intention lifts a function call out of an <b>if</b> expression.
```

Comment:
Please do not use `This intention ...`, for example see [AddJvmStaticIntention intention description](https://github.com/JetBrains/intellij-community/blob/master/plugins/kotlin/idea/resources-en/intentionDescriptions/AddJvmStaticIntention/description.html)

as well, please wrap code fragments with `<code>...</code>` rather than `<b>...</b>`

Reply:
95b29ed